app/controllers/ems_container_controller.rb

```diff
@@ -32,11 +32,6 @@ def ems_container_form_fields
     ems_form_fields
   end
 
-  def new
```

Comment:
This is an override that only removes atomic types from UI. 
I removed the classes and types entirely, so it is no longer necessary.